packages/frontend-shared/src/gql-components/ExternalLink.vue

```diff
@@ -23,10 +23,11 @@ import BaseLink from '../components/BaseLink.vue'
 import { useExternalLink } from '../gql-components/useExternalLink'
 
 const props = withDefaults(defineProps<{
-  href: string,
+  href?: string,
   useDefaultHocus?: boolean
 }>(), {
   useDefaultHocus: true,
+  href: '',
```

Comment:
Why was this change necessary? I can't find the usage change.
And a link without an href prop, I don't understand how it could happen.

Reply:
Yes. It was. It was throwing a warning. It happens when links are disabled.

Reply:
I'm also confused. If `href` is required, we shouldn't need a default value - the whole point is that it'll never use the default, since it's required in the first place.

Reply:
The link can be disabled, in which case, we pass null correctly. Previously, buttons that were externals and disabled were throwing warnings.

<img width="551" alt="Screen Shot 2022-02-28 at 12 57 45 PM" src="https://user-images.githubusercontent.com/2801156/156034019-47c78eaf-9f50-4c25-8473-b9e13b1445a8.png">